projects/plugins/jetpack/extensions/blocks/conversation/components/participants-controls.js

```diff
@@ -14,9 +14,7 @@ function ParticipantsLabelControl( { className, participants, onDelete } ) {
 		<div className={ `${ className }__participant-control` }>
 			{ participants.map( ( { label, slug } ) => (
 				<div key={ `${ slug }-key` } className={ `${ className }__participant` }>
-					<div className={ `${ className }__participant-label` }>
-						{ getPlainText( label ) }
-					</div>
+					<div className={ `${ className }__participant-label` }>{ getPlainText( label ) }</div>
```

Comment:
This and what follows are changes that were generated by Prettier when I updated the branch with latest `master`.